janusgraph-core/src/main/java/org/janusgraph/diskstorage/BackendTransaction.java

```diff
@@ -132,6 +134,10 @@ public void enableCache() {
         this.cacheEnabled = true;
     }
 
+    public boolean isCacheEnabled(){
```

Comment:
Is this being used anywhere?

Reply:
It's used in the added test and I also think it might be useful for anyone who wants to check if current transaction has cache enabled or not

Reply:
Did you forget to commit some code? I didn't see code logic that actually "disables" DB cache.

Reply:
I added a test called `testSkipDBCacheRead` into the PR. It sets skipDBCacheRead to `true` which disables cache for the current transaction. Then for the current transaction we use this method to double check that the cache is actually disabled